test.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+sudo docker build --force-rm -t pytrac/debian:stable .
+sudo docker run -rm -t -i  -v $PWD:/home/pytrac/data pytrac/debian:stable ./venv/bin/python ./setup.py test
```

Comment:
Please don't use sudo, as this is not required on OSX.

This is how we do in the buildbox:
https://github.com/Jimdo/buildbox#quick-start

Also, `set -e` ftw. :)